File: 김윤수/1주차/boj1924.cpp
#include <iostream>

int main(int argc, char* argv[]){
    int x, y;
    std::cin >> x >> y;
    int MonthToDay[13] = {0, 31, 28, 31,30, 31, 30, 31, 31, 30, 31, 30, 31};
    char *dayOfWeek[7] = { "SUN", "MON", "TUE", "WED", "THU", "FRI","SAT"};
    int day = y;
    for(int i = 1; i < x; i++)
        day += MonthToDay[i];
    std::cout << dayOfWeek[day % 7] ;
    return 0;

}

File: 김윤수/1주차/boj1924.py
monthToDay = [0,31,28,31,30,31,30,31,31,30,31,30,31]
dayOfWeek = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT']
x, y = input().split()
month = int(x)
day = int(y)

for i in range(1,month):
    day += monthToDay[i]
print(dayOfWeek[day%7])